src/markdown-script.ts

```diff
@@ -5,6 +5,71 @@ import { pintoraStandalone } from '@pintora/standalone'
 
 let hasThemeBeSet = false
 
+function formatError(error){
+  return `
+    <div>
+      <style>
```

Comment:
Can we move this style to an external `.css` file and import it only once through a `<link>` tag somewhere in the markdown preview page?
This will benifit both maintainance and performance.

Reply:
Sure! I'm working on it right now ^^

Reply:
Hello! I've made some changes with the code. It wouldn't let me to create an external .css file but other than that, I already fixed the concerns you asked me with the block collapsing. Thank you!

![image](https://user-images.githubusercontent.com/106539458/186998702-a9fd2f1a-dc43-4fe0-96c5-f57a541f62e2.png)

![image](https://user-images.githubusercontent.com/106539458/186998712-8899b915-cf36-466f-b688-a66465292e94.png)